batch/sql/fix-n-cancelled-creating-jobs.sql

```diff
@@ -201,15 +201,15 @@ select t.user,
        0,
        t.n
 FROM (
-  select batches.user, jobs.inst_coll, count(*) as n
-  from jobs
-  left join batches_cancelled
-  on jobs.batch_id = batches_cancelled.id
-  left join batches
-  on jobs.batch_id = batches.id
-  where jobs.state = 'Creating'
-    and (NOT jobs.always_run AND (jobs.cancelled OR (batches_cancelled.id is not null)))
-  group by batches.user, jobs.inst_coll
+  SELECT batches.user, jobs.inst_coll, count(*) AS n
```

Comment:
So this only updates the row with token = 0. What happens to the other rows? I think we need to set all other rows to 0.

Reply:
I do, the pervious statement in this transaction is an UPDATE that sets them all to 0.